### PR TITLE
[MEX-331] fees collector admin endpoints

### PR DESCRIPTION
**Pull request by claudiulataretu**
## Reasoning
- fees collector resolver missed some of admin endpoints
  
## Proposed Changes
- add known contracts update endpoints
- add known tokens update endpoints

### CODE REVIEW

File: src/modules/fees-collector/services/fees-collector.transaction.service.ts [hunk 1]
```diff
@@ -72,4 +79,50 @@ export class FeesCollectorTransactionService {
             .buildTransaction()
             .toPlainObject();
     }
+
```
Comment (danielailie) [inline src/modules/fees-collector/services/fees-collector.transaction.service.ts:82]: this one it does not do what it says, if remove is true then it removes the contract not adding it

File: src/modules/fees-collector/services/fees-collector.transaction.service.ts [hunk 2]
```diff
@@ -72,4 +79,50 @@ export class FeesCollectorTransactionService {
...
+            : contract.methodsExplicit.addKnownContracts(endpointArgs);
+
+        return interaction
+            .withGasLimit(gasConfig.feesCollector.addKnownContracts)
+            .withChainID(mxConfig.chainID)
+            .buildTransaction()
+            .toPlainObject();
+    }
+
+    async addKnownTokens(
```
Comment (danielailie) [inline src/modules/fees-collector/services/fees-collector.transaction.service.ts:107]: same here

**Review by cfaur09 — APPROVED** [review]

File: src/modules/fees-collector/services/fees-collector.transaction.service.ts (hunk 1 above)
Comment (danielailie) [reply src/modules/fees-collector/services/fees-collector.transaction.service.ts:82]: maybe a better name will be handleKnownContract and also for the mutation

Comment (danielailie) [reply src/modules/fees-collector/services/fees-collector.transaction.service.ts:82]: or have 2 different mutiations

Comment (claudiulataretu) [reply src/modules/fees-collector/services/fees-collector.transaction.service.ts:82]: done

File: src/modules/fees-collector/services/fees-collector.transaction.service.ts (hunk 2 above)
Comment (claudiulataretu) [reply src/modules/fees-collector/services/fees-collector.transaction.service.ts]: done

**Review by danielailie — APPROVED** [review]

**Review by danielailie — APPROVED** [review]

**Review by cfaur09 — APPROVED** [review]

**Pull request merged**